CI-API.Data/Repository/AdminRepository.cs

```diff
@@ -1106,6 +1106,178 @@ public async Task<JsonResult> DeleteStory(AdminPanelStoryViewModel storyData)
 
         #endregion
 
+        #region banner
+
+        #region GetAllBanners
+        public async Task<JsonResult> GetAllBanners(string? search)
+        {
+            try
+            {
+
+                if (search == null)
```

Comment:
Better to use string.isnullorEmpty and keep habit of when dealing with string

Reply:
changes done sir